internal/validator/pipeline.go

```diff
@@ -14,7 +15,39 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-func ValidatePipeline(ctx context.Context, p v1.Pipeline) error {
+// resolveParameterExpressions resolves Tekton parameter expressions like $(params.paramName) with runtime values
+func resolveParameterExpressions(input string, runtimeParams map[string]string) string {
+	// Match $(params.paramName) pattern
+	re := regexp.MustCompile(`\$\(params\.([^)]+)\)`)
```

Comment:
I think we can leverage pac to do this by adding the additional parameters to [this](https://github.com/konflux-ci/tektor/blob/db6caa2abda705df83d73d6b668ee85b7ff5e90f/internal/pac/pac.go#L61) map. That should make both `resolveParameterExpressions` and `resolveParamsInTaskRefParams` no longer needed.

Reply:
removed